examples/demo_dlmbl/solution.py

```diff
@@ -724,7 +878,7 @@ def min_max_scale(input):
 )
 trainer.fit(fluor2phase_model, datamodule=fluor2phase_data)
```

Comment:
Receiving an error: 
`ValueError: y_pred and y should have same shapes, got torch.Size([12, 2, 1, 256, 256]) and torch.Size([12, 1, 1, 256, 256]).`

Reply:
yep that was a typo on my side.